src/App.js

```diff
@@ -1,5 +1,6 @@
 import './App.css';
 import UserList1 from './userList1';
+import UserList2 from './userList2';
 
 function App() {
```

Comment:
Usually we don't use the funtion notation, and instead use arrow functions. Not sure why though.
```suggestion
const App = () => {
```

Reply:
👍